### PR TITLE
ufal/fe-download-bitstream-back-to-item

### PR DESCRIPTION
**Pull request by milanmajchrak**
| Phases            | MP | MM  |  MB  | MR |   JM | Total  |
|-----------------|----:|----:|----:|-----:|-----:|-------:|
| ETA                  |  0  |  0  |    0 |     0 |      0 |        0 |
| Developing      |  0  |  6  |    0 |    0 |      0 |         0 |
| Review             |  0  |  0  |    0 |    0 |      0 |         0 |
| Total                |   -  |   -  |   -   |  -    |   -    |         0 |
| ETA est.             |      |      |       |       |         |         0 |
| ETA cust.           |   -  |   -  |   -  |   -   |   -     |        0 |

### CODE REVIEW

**Comment by milanmajchrak** [comment]
There was vanilla problem, because the downloading has started before the downloading page was loaded.

File: src/aai/aai.js [hunk 1]
```diff
@@ -72,6 +72,11 @@
               case 'local':
                 // DiscoJuice.UI.setScreen(opts.localauth);
                 // jQuery('input#login').focus();
+                var expires = new Date();
+                expires.setTime(expires.getTime() + 1 * 24 * 60 * 60 * 1000);
```
Comment (vidiecan) [inline src/aai/aai.js]: add docs

File: src/aai/aai.js [hunk 2]
```diff
@@ -72,6 +72,11 @@
...
+                var cookieString = "dsRedirectUrl=" + getCookie('dsRedirectUrl') + ";expires=" + expires.toUTCString() + ";path=/";
```
Comment (vidiecan) [inline src/aai/aai.js]: is this really needed?

Comment (milanmajchrak) [reply src/aai/aai.js]: It is working without it, I will remove it, thank you

**Review by vidiecan — APPROVED** [review]

**Pull request merged**